.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: ci 
+on:
+  push:
+    branches:
+      - newdocs
```

Comment:
Change trigger to push on master and add cache.

Reply:
Wasn't sure how to add cache, but added master.

Reply:
This should work:
```
      - name: Setup caching
        uses: actions/cache@v3
        with:
          key: ${{ github.sha }}
          path: .cache
```
Check out: https://gaseri.org/en/blog/2022-11-01-publishing-material-for-mkdocs-website-to-github-pages-using-custom-actions-workflow/